daisy_workflows/image_import/enterprise_linux/translate.py

```diff
@@ -197,14 +198,17 @@ def yum_install(g, *packages):
   Raises:
       RuntimeError: If there is a failure during installation.
   """
-  try:
-    g.command(['yum', 'install', '-y'] + list(packages))
-  except Exception as e:
-    logging.debug('Failed to install {}. Details: {}.'.format(packages, e))
-    raise RuntimeError(
-        'Verify that you have specified the correct operating system '
-        'in the `--os` flag.  If you are bringing your own license (BYOL), '
-        'also verify that your subscription is eligible to be run on GCP.')
+  for i in range(6):
+    try:
+      time.sleep(i**2)
```

Comment:
Wouldn't this add delay even when not necessary, since it's before `yum install`? Shouldn't the sleep be in the `except` block?

Reply:
Since `i` is zero on the first iteration, there's no delay prior to the first call to yum.

Putting it in the except block incurs a large delay after the last call to yum, if all of the calls fail (assuming there's no extra code to determine if it's the last call).

Reply:
Add a comment that sleep is ignored the first time as it's not obvious

Reply:
sgtm